src/course/syllabus/apprenticeship/projects/in-house-build-2/spikes.md

```diff
@@ -0,0 +1,7 @@
+# Spikes
+
+There are no defined spikes for this week.
+
+You may find that you need to manage spikes within your sprints. Remember to follow [the techniques you followed earlier in the programme](../../../../handbook/spikes).
```

Comment:
```suggestion
You may find that you need to manage spikes within your sprints. Remember to follow [the techniques you followed earlier in the programme](/course/handbook/spikes).
```

It's probably a bit easier to reference much-higher-up paths like this from the root rather than having to figure out how many levels of `..` 😅

Reply:
Nice one, thanks. 😅